app/models/sitewide_setting.rb

```diff
@@ -12,4 +11,20 @@ def name
     key.humanize
   end
 
+  def self.set(key, on)
+    flag = find_by_key_or_create(key)
+    flag.update(on: on)
```

Comment:
Actually, instead of using the bang variant above, maybe better to rely on the fact that this will return false if the update fails and then update the rake task to complain if the setting could not be set.